api/src/integration/java/org/pmiops/workbench/AbsorbAcceptanceTest.java

```diff
@@ -46,21 +48,48 @@ public void testUserHasLoggedIntoAbsorb_False() throws Exception {
 
   @Test
   @Disabled("RW-11039")
-  public void testGetEnrollments() throws Exception {
+  public void testGetEnrollments_RTIncomplete() throws Exception {
+    // Setup:
+    // - The user has logged into Absorb
+    // - The user has not completed RT training
+    var enrollments = absorbService.getActiveEnrollmentsForUser(loggedInRTIncompleteUserEmail);
+    assertThat(enrollments.size()).isEqualTo(1);
+
+    var rtTrainingEnrollment =
+        enrollments.stream()
+            .filter(e -> e.courseId.equals(config.absorb.rtTrainingCourseId))
+            .findFirst();
+    assertThat(rtTrainingEnrollment.isPresent()).isTrue();
+    assertThat(rtTrainingEnrollment.get().completionTime).isNull();
+
+    var ctTrainingEnrollment =
+        enrollments.stream()
+            .filter(e -> e.courseId.equals(config.absorb.ctTrainingCourseId))
+            .findFirst();
+    assertThat(ctTrainingEnrollment.isPresent()).isFalse();
+  }
+
+  @Test
+  @Disabled("RW-11039")
+  public void testGetEnrollments_RTComplete() throws Exception {
     // Setup:
     // - The user has completed RT training in Absorb
     // - The user has not completed CT training in Absorb
-    var enrollments = absorbService.getActiveEnrollmentsForUser(partiallyCompleteUserEmail);
+    var enrollments = absorbService.getActiveEnrollmentsForUser(loggedInRTCompleteUserEmail);
+
+    // After completing RT training, it may take a few minutes to be auto-enrolled
+    // in CT training. In that case, this line fails.
+    // See https://docs.google.com/document/d/1ByiB1UVQDWHtRR2LOeNmvjEBbmqT7IRX2MVcwwRKA68
     assertThat(enrollments.size()).isEqualTo(2);
 
     var rtTrainingEnrollment =
         enrollments.stream()
             .filter(e -> e.courseId.equals(config.absorb.rtTrainingCourseId))
             .findFirst();
     assertThat(rtTrainingEnrollment.isPresent()).isTrue();
-    // Completed at 2:14PM EST on 2023-10-03
+    // Completed at 2:56PM EST on 2023-10-13
     assertThat(rtTrainingEnrollment.get().completionTime)
-        .isEqualTo(Instant.parse("2023-10-03T18:14:03.977Z"));
+        .isEqualTo(Instant.parse("2023-10-13T18:56:38.500Z"));
```

Comment:
I created a new `loggedInRTCompleteUserEmail` with the same state as the prior `partiallyCompleteUserEmail` to ensure backwards compatibility.